charms/reconciler.py

```diff
@@ -29,14 +30,9 @@ def reconcile(self, event):
 
         self.stored.reconciled = False
 
-        with status.context(self.charm.unit):
+        with status.context(self.charm.unit, self.exit_status):
             try:
-                result = self.reconcile_function(event)
+                self.reconcile_function(event)
                 self.stored.reconciled = True
-                return result
```

Comment:
The ops framework doesn't expect handlers to have return values... let's get rid of this